Ah, it was going so well! Until...
```
[ 25%] Building CXX object aseprite/third_party/cityhash/CMakeFiles/cityhash.dir/src/city.cc.obj
D:\a\qaseprite\qaseprite\aseprite\third_party\cityhash\src\city.cc:92:10: fatal error: byteswap.h: No such file or directory
   92 | #include <byteswap.h>
      |          ^~~~~~~~~~~~
compilation terminated.
```

And another, which should be easy to fix:
```
[ 18%] Building CXX object aseprite/laf/base/CMakeFiles/laf-base.dir/win/ver_query_values.cpp.obj
D:\a\qaseprite\qaseprite\aseprite\laf\base\win\ver_query_values.cpp: In function 'std::map<std::__cxx11::basic_string<char>, std::__cxx11::basic_string<char> > base::ver_query_values(const wchar_t*)':
D:\a\qaseprite\qaseprite\aseprite\laf\base\win\ver_query_values.cpp:48:15: error: 'uint8_t' was not declared in this scope
   48 |   std::vector<uint8_t> data(size);
      |               ^~~~~~~
D:\a\qaseprite\qaseprite\aseprite\laf\base\win\ver_query_values.cpp:20:1: note: 'uint8_t' is defined in header '<cstdint>'; did you forget to '#include <cstdint>'?
   19 | #include <windows.h>
  +++ |+#include <cstdint>
   20 | 
```

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -147,9 +147,13 @@ jobs:
         pushd aseprite
         patch -p1 < ../aseprite-fix-zlib-include.patch
         patch -p1 < ../aseprite-msvc-dynamic-runtime.patch
-        pushd laf
-        patch -p1 < ../../laf-msvc-dynamic-runtime.patch
         popd
+        pushd aseprite/laf
+        patch -p1 < ../../laf-msvc-dynamic-runtime.patch
+        patch -p1 < ../../laf-include-cstdint.patch
+        popd
+        pushd aseprite/third_party/cityhash
+        patch -p1 < ../../../cityhash-fix-byteswap.patch
         popd
         cmake -B build -DCMAKE_BUILD_TYPE=Release ${{ matrix.cmake_args }}
         cmake --build build --config Release

--- a/cityhash-fix-byteswap.patch
+++ b/cityhash-fix-byteswap.patch
@@ -1,0 +1,13 @@
+diff --git a/src/city.cc b/src/city.cc
+index 41cd5ee..a362d8c 100644
+--- a/src/city.cc
++++ b/src/city.cc
+@@ -47,7 +47,7 @@ static uint32 UNALIGNED_LOAD32(const char *p) {
+   return result;
+ }
+ 
+-#ifdef _MSC_VER
++#ifdef _WIN32
+ 
+ #include <stdlib.h>
+ #define bswap_32(x) _byteswap_ulong(x)

--- a/laf-include-cstdint.patch
+++ b/laf-include-cstdint.patch
@@ -1,0 +1,12 @@
+diff --git a/base/win/ver_query_values.cpp b/base/win/ver_query_values.cpp
+index 71df1a7..88944cf 100644
+--- a/base/win/ver_query_values.cpp
++++ b/base/win/ver_query_values.cpp
+@@ -12,6 +12,7 @@
+ 
+ #include "base/string.h"
+ 
++#include <cstdint>
+ #include <cstdio>
+ #include <string_view>
+ #include <vector>

